examples/cisco/demo_interface.pp

```diff
@@ -87,6 +87,16 @@
     ipv4_arp_timeout => 300,
   }
 
+  cisco_overlay_global { 'default':
+    anycast_gateway_mac                    => '1.1.1',
+  }
+
+  cisco_interface { 'vlan97':
```

Comment:
Does this make sense in demo_interface or would it be a better fit for demo_vxlan?